src/lib/core/style/focus-classes.ts

```diff
@@ -3,7 +3,10 @@ import {Observable} from 'rxjs/Observable';
 import {Subject} from 'rxjs/Subject';
 
 
-export type FocusOrigin = 'mouse' | 'keyboard' | 'program';
+const TOUCH_BUFFER_MS = 650;
```

Comment:
Also mind exporting this variable? So the tests can use it for their `setTimeout`'s?